docs/Operators.md

```diff
@@ -4943,6 +4983,163 @@ expect(node, inputs=[x], outputs=[y],
 </details>
 
 
+### <a name="Loop"></a><a name="loop">**Loop**</a>
+
+  Generic Looping construct. This loop has multiple termination conditions:
+  
+  1) Trip count. Iteration count specified at runtime. Set by
```

Comment:
With Scan being acceptable also, do we still need trip_count?

Reply:
It's a provided number of iterations (as a scalar tensor) vs picking the dim in Scan op

I think fixed trip count is useful - it saves from having to do add-1 manually. Also it'd make it easier if we want to translate ScanOp into generic Loop